packages/teleport/package.json

```diff
@@ -4,8 +4,8 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "g-start",
-    "build": "g-build --progress --bail",
+    "start": "g-start --config $(pwd)/webpack.dev.config.js",
```

Comment:
The `$(pwd)` shouldn't be necessary here.

Reply:
It's required, it's unable to find it otherwise and throws an error